### PR TITLE
change 'show search module text entry' and add tooltip.

### PR DESCRIPTION
**Pull request by darkelectron**
Change _show search module text entry_ to _show module groups and/or search text entry_ in the preferences window under `darkroom` submenu, to clarify what the setting does and add a tooltip.

### CODE REVIEW

**Review by TurboGit — CHANGES_REQUESTED** [review]
As we are at it, since it needs a restart you want to add the attribute:
 restart="true"

On the dtconfig node. This will trigger a message on screen for user.

File: data/darktableconfig.xml.in [hunk 1]
```diff
@@ -1009,8 +1009,8 @@
       </enum>
     </type>
     <default>show both</default>
-    <shortdescription>show search module text entry</shortdescription>
-    <longdescription/>
+    <shortdescription>show module groups and/or search text entry</shortdescription>
+    <longdescription>show module groups and/or search text entry. (needs a restart)</longdescription>
```
Comment (johnny-bit) [inline data/darktableconfig.xml.in:1013]: if it really needs restart, shouldn't you also change to
```xml
<dtconfig prefs="darkroom" restart="true">
```
couple lines above.

**Review by johnny-bit — COMMENTED** [review]
yeah, makes sense!

File: data/darktableconfig.xml.in (hunk 1 above)
Comment (darkelectron) [reply data/darktableconfig.xml.in:1013]: Thanks for the tip. Made the change.

**Review by TurboGit — APPROVED** [review]
Looks good to me now.

**Comment by TurboGit** [comment]
A side note: Please never merge master into your topic branch, for a working branch it is better to rebase and in this case even squash the work in a single commit as all changes are related. I'll squash & merge this one. Thanks.

**Pull request merged**